src/test/java/nl/praegus/fitnesse/responders/testHistory/TestHistoryRecentTest.java

```diff
@@ -2,9 +2,9 @@
 import org.junit.Test;
 
 import java.io.File;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestHistoryRecentTest {
```

Comment:
De naam van de klasse moet gelijk zijn aan de klasse die je test, + Test. In dit geval dus RecentTestHistoryTest

Reply:
De naam van de klasse is veranderd, zodat het overeenkomt met de naam van de klasse van de test.